packages/cli-e2e/__tests__/angular.specs.ts

```diff
@@ -121,16 +125,18 @@ describe('ui', () => {
 
     it('should send a search query when the page is loaded', async () => {
       await page.goto(searchPageEndpoint, {waitUntil: 'networkidle2'});
+      await page.waitForSelector(searchboxSelector);
```

Comment:
So, I noticed that sometimes, the expectation are run too soon (i.e. the project javascript didn't have time to run).
To avoid that, I put a waitFor on the searchboxSelector.

This allows us to ensure that the scripts has been loaded etc.
Worst case, it didn't, and the expectation will still faceplant.